registry.json

```diff
@@ -5218,5 +5218,19 @@
    "telegram": "VegasAda",
    "operatorName": "",
    "github": ""
+ },
+  {
+   "ticker": "SOL_1",
+   "poolId": "9adbf9ce3c2da566017898c4871126f1371def427c6e245fd7c02211",
+   "social": {
+     "twitter": "SolydarOrg",
+     "website": "https://www.solydar.org",
+      "facebook": "",  
+      "instagram": "",
+      "youtube": ""
+   },
+   "telegram": "",
```

Comment:
This field must be provided and must be the username of an individual and not a group or channel